src/makeCalculator.js

```diff
@@ -38,6 +38,33 @@
  */
 function makeCalculator() {
   // write code here
+  const calculator = {
+    result: 0,
+    operate(operation, x) {
+      operation(x);
+
+      return this;
+    },
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+    add(a) {
+      calculator.result += a;
+    },
+    subtract(a) {
+      calculator.result -= a;
+    },
+    multiply(a) {
+      calculator.result *= a;
+    },
+    divide(a) {
+      calculator.result /= a;
+    },
```

Comment:
You need to add a check for 0, because you cannot divide by 0 according to the rules of mathematics